### DISCUSSION TITLE
Mouse velocity?

### DISCUSSION THREAD

**Discussion by Link-Games** [TurboWarp/extensions#184] [Ideas]
Can someone make blocks that detect mouse velocity? (like mousevelx and mousevely)

**Comment by showierdata9978** [comment]
Yeah i feel like that would be easy :D

https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent

**Comment by GarboMuffin** [comment]
Can you elaborate what your use case is for this? Sure it's not technically hard to provide but we need to make sure if it makes sense to provide an extension for this. Note that you can already track velocity on your own with the existing blocks.

**Reply by showierdata9978** [reply]
yeah, it is incredibly easy to make it (made it in a few minutes lol: https://scratch.mit.edu/projects/800955059/)
